libraries/CurieBLE/src/internal/BLEDeviceManager.h

```diff
@@ -392,7 +400,7 @@ class BLEDeviceManager
     uint8_t    _peer_adv_data[BLE_MAX_ADV_BUFFER_CFG][BLE_MAX_ADV_SIZE];
     uint8_t    _peer_adv_data_len[BLE_MAX_ADV_BUFFER_CFG];
     uint8_t    _peer_scan_rsp_data[BLE_MAX_ADV_BUFFER_CFG][BLE_MAX_ADV_SIZE];
-    int8_t     _peer_scan_rsp_data_len[BLE_MAX_ADV_BUFFER_CFG];
+    uint8_t    _peer_scan_rsp_data_len[BLE_MAX_ADV_BUFFER_CFG];
```

Comment:
@SidLeung @sgbihu I think PR https://github.com/01org/corelibs-arduino101/pull/500 was reverted here. Is this change intended?